car: 
  length: 1

global_path: global_path

trajectory:
  lookahead: 30
  total_time: 5
  local_path: local_path
  vel_magnitude: 0.5
  
  
tracking:
  odom: /vesc/odom
  control: /vesc/high_level/ackermann_cmd_mux/input/default
  kp: 10.0
  kd: 3.0
  goal_threshold: 0.5
  